streaming/src/main/java/org/apache/carbondata/streaming/CarbonStreamRecordReader.java

```diff
@@ -115,7 +112,7 @@
 
   // vectorized reader
   private StructType outputSchema;
-  private ColumnarBatch columnarBatch;
```

Comment:
Change the class name to sparkStreamVectorized reader, move this to spark2 module